.circleci/config.yml

```diff
@@ -27,6 +27,14 @@ commands:
   build_and_test:
     description: "Run JVM tests"
     steps:
+      - run:
+          name: Get sbt binary
```

Comment:
From official guide: [CircleCI: Language Guide: Scala](https://circleci.com/docs/2.0/language-scala/).

Reply:
Also, in practice, this step is taking around 3/4 seconds to complete, so it doesn't look like a big drawback to install sbt in every run.